ansys/dpf/core/server_context.py

```diff
@@ -0,0 +1,101 @@
+"""
+ServerContext
+=============
+
+Gives the ability to choose the context with which the server should be started.
+The context allows to choose which capabilities are available.
+"""
+from enum import Enum
+
+
+class EContextType(Enum):
+    pre_defined_environment = 0
+    """DataProcessingCore.xml that is next to DataProcessingCore.dll/libDataProcessingCore.so will
+    be taken"""
+    premium = 1
+    """Gets the Specific premium DataProcessingCore.xml."""
+    user_defined = 2
+    """Load a user defined xml using its path."""
+    custom_defined = 3
+    """Loads the xml named "DpfCustomDefined.xml" that the user can modify."""
+    entry = 4
+    """Loads the minimum number of plugins for a basic usage."""
+
+
+class ServerContext:
+    """The context allows to choose which capabilities are available server side.
+
+    The default parameters can be overwritten using the DPF_SERVER_TYPE environment
```

Comment:
@cbellot000 It is not clear to me from this sentence the link between class ServerContext and the server types.

Reply:
sorry, bad copy paste ;)